internal/runner/rspec.go

```diff
@@ -14,18 +14,13 @@ import (
 	"github.com/kballard/go-shellquote"
 )
 
-// In future, Rspec will implement an interface that defines
-// behaviour common to all test runners.
-// For now, Rspec provides rspec specific behaviour to execute
-// and report on tests in the Rspec framework.
+var _ = TestRunner(Rspec{})
```

Comment:
Can you explain the use of this? I've never seen this before

Reply:
This is a build-time check rather than a run-time check to ensure that the Rspec struct implements the TestRunner interface :+1:

Reply:
oic TIL https://medium.com/@matryer/golang-tip-compile-time-checks-to-ensure-your-type-satisfies-an-interface-c167afed3aae